Resources/Prototypes/Entities/Objects/Tools/hand_labeler.yml

```diff
@@ -25,3 +25,6 @@
           - Item
         tags:
           - Structure
+    - type: Tag
+      tags:
+      - HandLabeler # DeltaV - fish labeler
```

Comment:
This tag already exists

https://github.com/DeltaV-Station/Delta-v/blob/6687a3187faaeac78153e17d1f4298365efda7a2/Resources/Prototypes/DeltaV/tags.yml#L18-L19

Reply:
Wait wrong file, meant to reply to Resources/Prototypes/Nyanotrasen/tags.yml